pkg/timeutils/utils_test.go

```diff
@@ -0,0 +1,24 @@
+package timeutils
+
+import (
+	"fmt"
+	"testing"
+	"time"
+)
+
+func TestGetTimestamp(t *testing.T) {
+	now := time.Now()
+	cases := []struct{ in, expected string }{
+		{"0", "-62167305600"},
+		{"2015-12-05", "1449273600"},
+		{"invalid", "invalid"},
+		{"1m", fmt.Sprintf("%d", now.Add(-1*time.Minute).Unix())},
```

Comment:
And based on https://github.com/docker/docker/pull/2348#issuecomment-27168600 I think we need to test lots of cases where we shorten the format - e.g.
```
2006-01-02T15:04:05.999999999Z+07:00
2006-01-02T15:04:05.999999999
2006-01-02T15:04:05.99999
2006-01-02T15:04:05
2006-01-02T15:04
2006-01-02T
```